micro-infra-spring-config/src/main/java/com/ofg/infrastructure/property/FileSystemLocator.java

```diff
@@ -77,7 +78,12 @@ public ConfigLocations getConfigLocations() {
     private Map<String, Object> decrypt(Map<String, Object> sourceMap) {
         final Map<String, Object> result = new HashMap<String, Object>();
         for (Map.Entry<String, Object> entry : sourceMap.entrySet()) {
-            result.put(entry.getKey(), decryptIfEncrypted(entry.getValue()));
+            try {
+                result.put(entry.getKey(), decryptIfEncrypted(entry.getValue()));
+            } catch (Exception e) {
+                log.error(String.format("Exception occurred while trying to decrypt key [%s]", entry.getKey()), e);
```

Comment:
Yes, please skip the last parameter to avoid printing stack trace twice. BTW more idiomatic SLF4J is:

    log.error("Exception occurred while trying to decrypt key [{}]", entry.getKey());